setup.py

```diff
@@ -55,7 +55,7 @@
     "sentence-transformers",
     "open_clip_torch>=2.26.1",
     "peft",
-    "datasets[audio]>=1.4.0,<4.0.0",
+    "datasets[audio]>=1.4.0",
```

Comment:
Seems like there are other issues caused by datasets 4.0, namely torch 2.7 being required.
```suggestion
    "datasets[audio]>=1.4.0,<4.0.0",
```